viewer/packages/camera-manager/src/Flexible/FlexibleCameraMarkers.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { Mesh, MeshBasicMaterial, SphereGeometry, Scene, Object3D, Vector3 } from 'three';
+import { FlexibleControlsType } from './FlexibleControlsType';
+import { FlexibleCameraManager } from './FlexibleCameraManager';
+
+export class FlexibleCameraMarkers {
+  private readonly _scene?: Scene;
+  private _pivotMarker: Object3D | undefined;
+  private _lookAtMarker: Object3D | undefined;
+
+  //================================================
+  // CONSTRUCTOR
+  //================================================
+
+  constructor(scene: Scene) {
+    this._scene = scene;
+  }
+
+  //================================================
+  // INSTANCE METHODS: Update helper Objects
+  //================================================
+
+  public update(manager: FlexibleCameraManager): void {
+    if (this._scene === undefined) {
+      return;
+    }
+    const show = manager.controls.controlsType !== FlexibleControlsType.FirstPerson;
+    if (show && manager.options.showTarget) {
+      if (!this._pivotMarker) {
+        this._pivotMarker = this.createPivotMarker();
+        this._scene?.add(this._pivotMarker);
+      }
+      this.setPosition(this._pivotMarker, manager.controls.getTarget(), manager);
+    } else {
+      if (this._pivotMarker) {
+        this._scene?.remove(this._pivotMarker);
+        this._pivotMarker = undefined;
+      }
+    }
+    if (show && manager.options.showLookAt) {
+      if (!this._lookAtMarker) {
+        this._lookAtMarker = this.createLookAtMarker();
+        this._scene?.add(this._lookAtMarker);
+      }
+      this.setPosition(this._lookAtMarker, manager.controls.getLookAt(), manager);
+    } else {
+      if (this._lookAtMarker) {
+        this._scene?.remove(this._lookAtMarker);
+        this._lookAtMarker = undefined;
```

Comment:
Can remove question marks after `this._scene`

Reply:
b7e7709b726354b0fe1e97fc6dbfa502014f9bb2 and f89f68d3f183bbc444bcd44616d3bed204d6d30c